src/systems/filecoin_vm/runtime/runtime.id

```diff
@@ -95,13 +97,14 @@ type Runtime interface {
     // Computes an address for a new actor. The returned address is intended to uniquely refer to
     // the actor even in the event of a chain re-org (whereas an ID-address might refer to a
     // different actor after messages are re-ordered).
+    // Always an ActorExec address.
     NewActorAddress() addr.Address
 
     // Creates an actor in the state tree, with empty state. May only be called by InitActor.
     CreateActor(
         // The new actor's code identifier.
         codeId   actor.CodeID
-        // Address under which the new actor's state will be stored.
+        // Address under which the new actor's state will be stored. Must be an ID-address.
```

Comment:
You should also be able to pass in a key address if the actor has one

Reply:
No, this cannot be used for creating account actors, which is why I made it explicit.